modules/serverless-pingsource.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-kn-source.adoc
+
+[id="serverless-pingsource_{context}"]
+= Using PingSource
+
+PingSource is used to periodically send ping events with a constant payload to an event consumer. It can be used to schedule sending events, like a timer, as shown in the example:
+
+.Example PingSource YAML
+----
+apiVersion: sources.knative.dev/v1alpha2
+kind: PingSource
+metadata:
+  name: test-ping-source
+spec:
+  schedule: "*/2 * * * *" <1>
+  jsonData: '{"message": "Hello world!"}' <2>
+  sink: <3>
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+----
+
+<1> The schedule of the event specified using https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#schedule[CRON expression].
+<2> The event message body expressed as a JSON encoded data string.
+<3> These are the details of the event consumer. In this example, we are using a Knative service named `event-display`.
+
+== Procedure
+. To verify that `PingSource` is working, create a simple Knative
+Service that dumps incoming messages to its log.
++
+Copy the example YAML into a file named `service.yaml`:
++
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display
```

Comment:
@abrennan89 let's use this

```
quay.io/openshift-knative/knative-eventing-sources-event-display
```

that is _exactly_ that image, but built via our own CI, than sync'd to quay.io